jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -351,35 +330,67 @@ private ContainerConfigurationTemplate pullContainerConfigJson(
   }
 
   /**
-   * Retrieves the cached base image.
+   * Retrieves the cached base images. If a base image reference is not a manifest list, returns a
+   * single image (if cached). If a manifest list, returns all the images matching the configured
+   * platforms in the manifest list but only when all of the images are cached.
```

Comment:
Why do we make sure to return the images matching the configured platforms in the manifest list only if *all* the images are cached?

Reply:
If we have cached only a subset of what we really need (i.e., Jib needs build, says, three `Image`s, but the cache has only two `Image`s), then it's not a cache hit as a whole, thus a cache miss.

Theoretically you could say, Jib could re-download only the missing `Image`s, but note that performance is not the reason for caching. It's about whether we can completely avoid accessing the Internet or not. So, if any image is missing in the cache, I am saying "cache miss."